apps/src/templates/ChallengeDialog.jsx

```diff
@@ -109,7 +109,7 @@ const ChallengeDialog = Radium(React.createClass({
             ...(this.props.complete ? styles.bannerComplete : {})
           }}
         >
-          <h1 style={styles.title}>
+          <h1 style={styles.title} id="challengeTitle">
```

Comment:
Tiny nit: Brent has been asking to prefix these with `uitest-` when they're only used for UI test.  I think so it's clear there isn't any JS or CSS that depends on this ID.